libmscore/layout.cpp

```diff
@@ -3857,6 +3861,7 @@ void Score::doLayoutRange(int stick, int etick)
 
       lc.prevMeasure = 0;
 
+      fixTicks();
```

Comment:
Why do we need it here?

Reply:
We need to reset applied tempo. Because tempo is setted in layout. And if we don't reset tempo new timeStretch is applied to the last applied timeStretch. It lead to twice timeStretch, next triple e.t.c

Reply:
It is worth adding this info to comment before calling this method in code.

Reply:
Done